docs/TpchBenchmark.md

```diff
@@ -0,0 +1,453 @@
+# Velox I/O Optimizations and the TpchBenchmark
```

Comment:
@mbasmanova / @pedroerp - This is a doc that summarizes the presentation we had in the Velox OSS monthly, and should help customers tune Velox for better IO performance. Please suggest if there is a different place you'd rather have it published.

Reply:
Thanks for putting this together! Made a few comments below.

Reply:
We could simply call this "I/O Optimizations and the TpchBenchmark" - the "Velox" part is implied.